test/infra/cchost.py

```diff
@@ -182,27 +182,22 @@ async def _start_process(self) -> None:
             cchost_env["FAKETIME_NO_CACHE"] = "1"
 
         LOG.debug("Starting cchost process...")
+        stdout_file = open(f"{self.workspace}/std.out", "w")
+        stderr_file = open(f"{self.workspace}/std.err", "w")
```

Comment:
Wouldn't these files be overwritten if cchost is restarted by the tests during the same pytest run? It could make sense to write these files in unique places (e.g., with a timestamp or test ID in the file name) to ensure that the captured logs can be referenced to a specific test.

Reply:
Yes, CCF tests create per-test and per-node directories for that exact reason, that's definitely something SCITT tests should do too.

Reply:
It's controlled by the fixtures. IIUC, currently all tests except marked as @isolated_test reuse the same setup, and so they share the log.

Isolated have their own `workspace` paths, so they don't interfere.

Reply:
You might want to change that and have more isolated tests, but I think it out of the scope of this PR.